webapp/portlet/src/main/resources/locale/portlet/social/ProfileWorkExperience_en.properties

```diff
@@ -11,6 +11,7 @@ profileWorkExperiences.endDate=End Date
 profileWorkExperiences.stillInPosition=Still in position
 profileWorkExperiences.removeExperience=Remove Experience
 profileWorkExperiences.formValidationError=Please check validation information in fields.
+profileWorkExperiences.invalidOrganizationName=The length of the text field "Organization" must be between "0" and "250".
```

Comment:
Please avoid using quotes and static values in resource bundles

Reply:
I am already intending to update it.
PS: the PR is not finished yet.